tests/libres_tests/res/enkf/test_local_config.py

```diff
@@ -75,7 +75,7 @@ def test_local_obs_data(self):
             self.assertTrue(node_added)
 
             # Error when adding existing obs node
-            with self.assertRaises(KeyError):
+            with self.assertRaises(Exception):
```

Comment:
Is this a base `Exception` or a specific one?

Reply:
Should be `KeyError` - did not know how to throw that from C++; got a tip from Zohar.